opentype-shaping-gurmukhi.md

```diff
@@ -1413,7 +1470,11 @@ typographically problematic.
 ### 6: Applying remaining positioning features from GPOS ###
 
 In this stage, mark positioning, kerning, and other GPOS features are
-applied. As with the preceding stage, the order in which these
+applied.  In preparation for this stage, glyph sequences should be
+flagged for possible application of GSUB features in stage 2,
```

Comment:
Probable typo: GSUB -> GPOS.

As an aside, I don't recall there being an Indic script which required sequences to be flagged for GPOS. Is this deliberately being kept generic, as other non-Indic scripts might require GPOS feature flagging?

Reply:
My first pass through here said "GSUB and GPOS" in all of these 'in-preparation-for' sentences, to keep them generic, before I realized that that was entirely unnecessary. Probably in my imagination I was thinking about something letter-to-letter like `dist` or a pre-base-reordering case, but you're right, this can go.